words/predicates.txt

```diff
@@ -820,21 +919,25 @@ road
 roan
 roasted
 robust
+rocky
 rogue
 romantic
 roomy
 root
 rose
 rough
 round
+rounded
 rowan
 royal
 ruby
 ruddy
+rumbling
 rune
 rural
 rust
 rustic
+rusty
```

Comment:
there are many forms of sexual slang of the format `rusty-[X]`. i really think we should get rid of this one